staging/src/k8s.io/pod-security-admission/admission/admission.go

```diff
@@ -227,18 +227,19 @@ func (a *Admission) Validate(ctx context.Context, attrs api.Attributes) *admissi
 // and checks existing pods in the namespace for violations of the new policy when updating the enforce level on a namespace.
 // The returned response may be shared between evaluations and must not be mutated.
 func (a *Admission) ValidateNamespace(ctx context.Context, attrs api.Attributes) *admissionv1.AdmissionResponse {
+	logger := klog.FromContext(ctx)
```

Comment:
`logger` is only used in error paths. Instead of paying the price for this call here on all calls to `ValidateNamespace`, better repeat it whenever needed.

Reply:
Makes sense. 

Done.